models/map.model.js

```diff
@@ -0,0 +1,16 @@
+const db = require("../db/connection");
+const fs = require("fs/promises");
+
+exports.selectJSON = () => {
+    return fs.readFile('endpoints.json', 'utf8')
+    .then((data) => {
+        // Do something with the data]
+        parsedData = JSON.parse(data)
+        return parsedData;
+    })
+    .catch((err) => {
+        return Promise.reject({ status: 404, message: "File does not" })
+
+    });
```

Comment:
I like the idea of this being here, but having a .catch in a model file isnt best practice. 

This also should not be a 404 but rather a 500 error as this would be an error on the server end for not having/being able to read the endpoints.json file.